test/cases/workable.test.sh

```diff
@@ -3,7 +3,7 @@
 # Waiting for Clusterpedia to be ready
 function ready() {
     local i
-    for ((i = 0; i < 300; i++)); do
+    for ((i = 0; i < 10; i++)); do
```

Comment:
github action performance is not very good, from 300 directly down to 10, whether it becomes prone to failure.

Let's go down to 200 or 150 first?